README.rst

```diff
@@ -99,6 +99,10 @@ and run the following ::
 
         pip install .
 
+In ``diffpy.structure>=3.3.1``, please ensure the dependency ``pycifrw`` is installed from ``conda-froge`` ::
```

Comment:
Please do a bit more checking.  Are the PyPI and conda version numbers the same?  if not, then it could be a version number issue. and the instructions would be to install `pycifrw>=???` or `pycifrw<=??`.  If the version is the same on PyPI and conda-forge, let the users know that explicitly (the conda-forge one works but the PyPI one doesn't).

We can also check to see if we can update our tests so it passes with both versions too.  

Let's discuss it, but we can also propose to make an issue/PR at pycifrw.